### ISSUE TITLE
SHA1Sum files have invalid format for use with CLI  `sha1sum`

### ISSUE THREAD

**Issue by omad** [opendatacube/odc-stats#40]
The `.sha1` checksum files generated by `odc-stats` are rejected by both the GNU and Busybox versions of `sha1sum -c`.

These tools expect the filename and checksum to be separate by one or two space characters, whereas `odc-stats` uses a tab (`\t`) character.

```
$ ls
ga_ls_tc_pc_cyear_3_x08y25_2015--P1Y_final.odc-metadata.yaml  ga_ls_tc_pc_cyear_3_x08y25_2015--P1Y_final_green_pc_10.tif
ga_ls_tc_pc_cyear_3_x08y25_2015--P1Y_final.proc-info.yaml     ga_ls_tc_pc_cyear_3_x08y25_2015--P1Y_final_green_pc_50.tif
ga_ls_tc_pc_cyear_3_x08y25_2015--P1Y_final.sha1               ga_ls_tc_pc_cyear_3_x08y25_2015--P1Y_final_green_pc_90.tif
ga_ls_tc_pc_cyear_3_x08y25_2015--P1Y_final.stac-item.json     ga_ls_tc_pc_cyear_3_x08y25_2015--P1Y_final_thumbnail.jpg
ga_ls_tc_pc_cyear_3_x08y25_2015--P1Y_final_bright_pc_10.tif   ga_ls_tc_pc_cyear_3_x08y25_2015--P1Y_final_wet_pc_10.tif
ga_ls_tc_pc_cyear_3_x08y25_2015--P1Y_final_bright_pc_50.tif   ga_ls_tc_pc_cyear_3_x08y25_2015--P1Y_final_wet_pc_50.tif
ga_ls_tc_pc_cyear_3_x08y25_2015--P1Y_final_bright_pc_90.tif   ga_ls_tc_pc_cyear_3_x08y25_2015--P1Y_final_wet_pc_90.tif

$ sha1sum -cw ga_ls_tc_pc_cyear_3_x08y25_2015--P1Y_final.sha1
sha1sum: invalid format
sha1sum: invalid format
sha1sum: invalid format
sha1sum: invalid format
sha1sum: invalid format
sha1sum: invalid format
sha1sum: invalid format
sha1sum: invalid format
sha1sum: invalid format
sha1sum: invalid format
sha1sum: invalid format
sha1sum: invalid format
sha1sum: invalid format
sha1sum: WARNING: 13 of 13 computed checksums did NOT match

$ sha1sum --version
sha1sum: unrecognized option '--version'
BusyBox v1.34.1 () multi-call binary.

Usage: sha1sum [-c[sw]] [FILE]...

Print or check SHA1 checksums

	-c	Check sums against list in FILEs
	-s	Don't output anything, status code shows success
	-w	Warn about improperly formatted checksum lines
```

This should be a really easy fix at https://github.com/opendatacube/odc-stats/blob/c30a71ecd462f2227b4d7dcd062b62feac88d6dd/odc/stats/io.py#L80

Thanks for reporting @jmettes !

**Comment by emmaai** [comment]
I encounter this too in the data test. Will fix it now.

**Comment by emmaai** [comment]
fixed by pr #62
